Doctrine/RouteConditionMetadataListener.php

```diff
@@ -49,6 +49,11 @@ public function loadClassMetadata(LoadClassMetadataEventArgs $eventArgs)
         }
 
         $meta = $eventArgs->getClassMetadata();
+
+        if (null === $meta->getReflectionClass()) {
```

Comment:
I don't know what is the condition to get the empty medatadata. 
I will investigate a bit more.

I see KnpLabs/DoctrineBehaviors doing the same condition check.